arcgis-ios-sdk-samples/Geometry/List transformations/ListTransformationsViewController.swift

```diff
@@ -0,0 +1,190 @@
+// Copyright 2018 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class ListTransformationsViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+
+    @IBOutlet var mapView: AGSMapView!
+    @IBOutlet var tableView: UITableView!
+    @IBOutlet var orderByMapExtent: UISwitch!
+    
+    var datatumTransformations = [AGSDatumTransformation]()
+    var defaultTransformation: AGSDatumTransformation?
+    let graphicsOverlay = AGSGraphicsOverlay()
+    var originalGeometry = AGSPoint(x: 538985.355, y: 177329.516, spatialReference: AGSSpatialReference(wkid: 27700))
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Add the source code button item to the right of navigation bar.
+        (self.navigationItem.rightBarButtonItem as! SourceCodeBarButtonItem).filenames = ["ListTransformationsViewController"]
+
+        // Get MapView from layout and set a map into this view
+        mapView.map = AGSMap(basemap: .lightGrayCanvasVector())
+        mapView.graphicsOverlays.add(graphicsOverlay)
+        
+        //add original graphic to overlay
+        addGraphic(originalGeometry, color: .red, style: .square)
+        
+        mapView.map?.load() { [weak self] (error) in
+            if error != nil {
+                print("map load error = \(String(describing: error))")
```

Comment:
Using `String.init(describing:)` is only necessary to prevent a warning because `error` is `Optional`. But you've already checked that `error` is not `nil`. Why not unwrap it and avoid it altogether?

```swift
if let error = error {
    print("map load error = \(error)")
```

Reply:
Done.